### ISSUE TITLE
model.plot() not working

### ISSUE THREAD

**Issue by Scipiock** [derrynknife/SurPyval#44]
Hello, I am new to this library. I have been looking for one that handles interval censored data and came across it. 
I quite like the Surpyval however I have not been able to visualize anything yet as i keep getting this and and empty plot:

"ValueError: keyword grid_b is not recognized; valid keywords are ['size', 'width', 'color', 'tickdir', 'pad', 'labelsize', 'labelcolor', 'labelfontfamily', 'zorder', 'gridOn', 'tick1On', 'tick2On', 'label1On', 'label2On', 'length', 'direction', 'left', 'bottom', 'right', 'top', 'labelleft', 'labelbottom', 'labelright', 'labeltop', 'labelrotation', 'grid_agg_filter', 'grid_alpha', 'grid_animated', 'grid_antialiased', 'grid_clip_box', 'grid_clip_on', 'grid_clip_path', 'grid_color', 'grid_dash_capstyle', 'grid_dash_joinstyle', 'grid_dashes', 'grid_data', 'grid_drawstyle', 'grid_figure', 'grid_fillstyle', 'grid_gapcolor', 'grid_gid', 'grid_in_layout', 'grid_label', 'grid_linestyle', 'grid_linewidth', 'grid_marker', 'grid_markeredgecolor', 'grid_markeredgewidth', 'grid_markerfacecolor', 'grid_markerfacecoloralt', 'grid_markersize', 'grid_markevery', 'grid_mouseover', 'grid_path_effects', 'grid_picker', 'grid_pickradius', 'grid_rasterized', 'grid_sketch_params', 'grid_snap', 'grid_solid_capstyle', 'grid_solid_joinstyle', 'grid_transform', 'grid_url', 'grid_visible', 'grid_xdata', 'grid_ydata', 'grid_zorder', 'grid_aa', 'grid_c', 'grid_ds', 'grid_ls', 'grid_lw', 'grid_mec', 'grid_mew', 'grid_mfc', 'grid_mfcalt', 'grid_ms']"

Is anyone else experiencing this?

**Comment by leester1690** [comment]
Exactly the same for me. This is probably coming from a subsequent matplotlib update where the grid_b keyword has been deprecated. I suppose this is why we have virtual environments to avoid problems like this. I don't know what the replacement is for grid_b yet, but if/until surpyval is updated, you'll need to adjust the package's source code.

**Comment by leester1690** [comment]
Exactly the same for me. This is probably coming from a subsequent matplotlib update where the grid_b keyword has been deprecated. I suppose this is why we have virtual environments. I don't know what the replacement is for grid_b yet, but if/until surpyval is updated, you'll need to adjust the package's source code.

**Comment by leester1690** [comment]
By the way, the reason I stumbled on surpyval is that reliability doesn't handle interval-censored data, which for my applications is necessary. surpyval is linked in the reliability docs. I can tell you that the Gumbel (SEV) example for multi-censored data in surpyal does exactly match what I generated in JMP, so I know surpyval is accurate for at least that example. Bummer I can't plot it though.

**Comment by Zjiangliuer** [comment]
same issue.

**Comment by ThunderBolt84** [comment]
I also have the same issue, and I was able to work around it to get the [quickstart](https://surpyval.readthedocs.io/en/latest/Quickstart.html) to run. 

I noticed that the package says it's supported from Python 3.6-3.9, so I installed 3.9 from [here](https://www.python.org/downloads/release/python-3913/) (Windows). I then checked versions of matplotlib until I found one that did not have the error with `py -3.9 "c:/path/to/code.py". 

matplotlib 3.6 does not have the `grid_b` issue, 3.7 does. To get the plot to actually display, I had to run `plt.show()` after `model.plot()`

Incidentally, I also found this library because it could handle interval-censored data. It's by far the best Python library I could find for probability plotting, but this issue hobbles it. 

TL;DR
1. Install Python 3.9
2. Install MatPlotLib 3.6
3. Run `plt.show` after `model.plot` (with `import matplotlib.pyplot as plt` obviously)
4. profit

**Comment by derrynknife** [comment]
fixed in 0.11.0
